### ISSUE TITLE
the accuracy is very bad

### ISSUE THREAD

**Issue by kwea123** [lktidaohuoxing/SiTPose#2]
any link to the paper?
I tried on own dataset, but the result is very bad compared to ground truth. Like if I input two identical images, the relative pose is far from identity

**Comment by lktidaohuoxing** [comment]
Sorry for the late reply, I have been busy looking for a job recently.
1. The training data does not include the same picture indeed.
2. For relative pose estimation, generalization is a big problem. I have reproduced many methods such as RPNet, RelPoseNet, and 8-points.  Both of these methods struggle in processing domain shift data.
My email is playtomarslk@gmail.com. Maybe you want to combine relative pose estimation with Nerf. Welcome to communicate with me.
